core/eolearn/core/eodata.py

```diff
@@ -435,6 +408,7 @@ def __contains__(self, key: object) -> bool:
             "`(feature_type, feature_name)` pairs."
         )
 
+    @deprecated_function(EODeprecationWarning, "Use the `merge` method instead.")
```

Comment:
When was this deprecated? Perhaps it could be thrown out.

Reply:
yesterday :D
I've never seen `patch1 + patch2` used in actual code and I'd kinda prefer not to